kedro/framework/cli/starters.py

```diff
@@ -62,6 +142,69 @@ def _remove_readonly(func: Callable, path: Path, excinfo: Tuple):  # pragma: no
     func(path)
 
 
+def _is_valid_starter_entrypoint_config(
+    module_name: str, config: KedroStarterSpec
```

Comment:
```suggestion
    module_name: str, spec: KedroStarterSpec
```
I like spec 🙂  `config` is already overused in kedro so let's try to avoid it here and call it a specification throughout.